Sources/ApodiniMigration/ApodiniMigratorInterfaceExporter.swift

```diff
@@ -208,7 +208,7 @@ final class ApodiniMigratorInterfaceExporter: InterfaceExporter {
             }
         }
         
-        if exportOptions.directory == nil && exportOptions.endpoint == nil {
+        if exportOptions.directory == nil && exportOptions.directory == nil {
```

Comment:
If with twice the same condition?

Reply:
Thanks, that is a mistake, I will fix this and merge the PR after that 👍